nas_spec/components/schemas/issuing/card-management/cardholders/properties/IssuingCardholder.yaml

```
@@ -37,29 +37,29 @@ properties:
     description: The cardholder's email.
     $ref: '#/components/schemas/IssuingEmail'
   phone_number:
-    description: The cardholder's phone number. This information will be used for authorisations requiring a 3DS challenge.
+    description: The cardholder's phone number. This information will be used for authorizations requiring a <a href="https://www.checkout.com/docs/risk-management/3d-secure">3DS challenge</a>.
     $ref: '#/components/schemas/IssuingPhoneNumber'
   date_of_birth:
     type: string
     format: date
-    description: The cardholder date of birth in the YYYY-MM-DD format.
+    description: The cardholder's date of birth, in the format `YYYY-MM-DD`.
     example: "1985-05-28"
   billing_address:
-    description: The  cardholder's billing address.
+    description: The cardholder's billing address.
     $ref: '#/components/schemas/IssuingAddress'
   residency_address:
-    description: The cardholder's residency address. If not passed the billing address will be used.
+    description: The cardholder's residency address. If this value is not provided, the cardholder's billing address will be used by default instead.
     $ref: '#/components/schemas/IssuingAddress'
   reference:
     $ref: '#/components/schemas/IssuingReference'
   account_entity_id:
-    description: The id of the entity that contains the currency account for the cardholder.
+    description: The ID of the entity that owns the cardholder's currency account.
```

```suggestion
    description: The ID of the entity that holds the cardholder's currency account.
```

Just a suggestion as the entity here is effectively the account holder (i.e. you can get the account balance from the entity id).